pyvista/plotting/renderer.py

```diff
@@ -1320,6 +1334,14 @@ def add_box_axes(
     def hide_axes(self):
         """Hide the axes orientation widget.
 
+        See Also
+        --------
+        show_axes
+            Show the axes orientation widget.
+
+        axes_enabled
+            Check if the axes orientation widget is enabled.
```

Comment:
Seems the `axes_enabled` link is not working.
![image](https://github.com/pyvista/pyvista/assets/7513610/a0ff7d3a-aeb1-41c8-9be7-75147b6d41fc)

Reply:
> Seems the `axes_enabled` link is not working.

Still note working at 4b40c0e96132b3f35681fc900265128a63533431.

Reply:
My bad 🤦 I was looking at the `Plotter` class `hide_axes`. I should see  `Renderer` class one. It works fine :)
It seems the link is not valid because there is no `axes_enabled` in the `Plotter` class. I will add it in another PR.